blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/serializers/JSONBasicRollupsOutputSerializer.java

```diff
@@ -169,8 +169,8 @@ private JSONObject getFilteredStatsForRollup(Rollup rollup, Set<MetricStat> filt
                 } else {
                     filteredObject.put(stat.toString(), filteredValue);
                 }
-            } catch (Exception ex) {
-                log.warn(ex.getMessage(), ex);
+            } catch (UnsupportedOperationException ex) {
+                log.warn(ex.getMessage());
```

Comment:
Should we log this as DEBUG?  Do we care if they ask for unsupported metrics?  that's user error, right?

Also, we can capture the 4XX in our repose access logs.